src/main/java/org/fundacionjala/coding/norman/banck_ocr/HistoryOneBankOcr.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.norman.banck_ocr;
+
+import java.util.List;
+
+/**
+ * Created by NORMAN on 2/7/2017.
+ */
+public class HistoryOneBankOcr extends FileNumberBankOcr {
```

Comment:
@AT-03/at-03 
We don't here inheritance 
Please have a single class for BankOCR